build/policies.c

```diff
@@ -298,6 +299,9 @@ static rpmRC processPolicies(rpmSpec spec, Package pkg, int test)
     free(name);
     free(types);
 
+    if (optCon)
```

Comment:
Don't bother checking for the NULL, poptFreeContext() does it anyway.

Reply:
done